app/models/customization_script.rb

```diff
@@ -2,5 +2,5 @@ class CustomizationScript < ApplicationRecord
   include NewWithTypeStiMixin
 
   acts_as_miq_taggable
-  belongs_to :provisioning_manager
+  belongs_to :ext_management_system, :foreign_key => "manager_id"
```

Comment:
Can this (and the corresponding migration) be split into separate PRs?  They're not really related to the rest of the changes here.

Also, would you mind deprecating the old `belongs_to` with:
```ruby
deprecate_belongs_to(:provisioning_manager, :ext_management_system)
```

Reply:
@bdunne I tested this change, and it looks like the deprecate is causing a problem. I am getting the following error in the UI:
undefined method `deprecate_belongs_to' for #Class:0x007f3ac1a98ff0 Did you mean? deprecate_constant [ems_physical_infra/show_list]

Reply:
Oh you'll also need to `include DeprecationMixin` in this class in order to use `deprecate_belongs_to`